.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,51 @@
+---
+name: 🐞 Bug Report
+about: Something went awry and you'd like to tell us about it.
+
+---
+
+<!--
+  ⚡️ katchow! We 💛 issues.
+
+  Please - do not - remove this template.
+  Please - do not - skip or remove parts of this template.
+  Or your issue may be closed.
+
+  👉🏽 Need help or tech support? Please don't open an issue!
+  Head to https://gitter.im/rollup/rollup or https://stackoverflow.com/questions/tagged/rollupjs
+
+  ❤️ Rollup? Please consider supporting our collective:
+  👉 https://opencollective.com/rollup/donate
+-->
+
+- Operating System:
+- Node Version:
+- NPM Version:
+- Rollup Version:
+- ${package} Version:
+
+### Expected Behavior
+
+
+### Actual Behavior
+
+
+### Code
```

Comment:
I think this section should be reworked a little to promote the ways of reporting an error that help us the most.

1. If the issue has to do with faulty code (many issues do) and can be replicated completely in the REPL, then providing a REPL link is usually the best option. A REPL link can easily be checked by any of us with minimal effort and quickly allows us to judge the severity of an issue.
2. If it cannot be replicated in the REPL, then a repo that exposes the issue is definitely the best option and this should be communicated. There are many situations where people post their rollup configs and hope we figure out what is wrong while the actual problem is caused by something in their code—which we do not have access to.
3. Only if the options above are not possible—sometimes code cannot be made public—people should post a rollup config and code samples or an incomplete gist. These kinds of issues are the hardest to track down and tend to lie around for some time.

Reply:
Makes sense to me. That approach seems much more streamlined and offloads more of the burden of proof onto the user up front. I'll poke at this.